.gitignore

```diff
@@ -14,20 +18,16 @@ __pycache__/
 .valgrindrc
 .vscode
 .favorites.json
-/*-*-*-*/
-/*-*-*/
```

Comment:
I was using this to have dirs git would ignore :(
However, it looks like I can maybe use the `#` prefix instead?

Reply:
Just edit your `.git/info/exclude` to ignore whatever you want.

Reply:
I keep forgetting why I don't use that more often.